### DISCUSSION TITLE
[Question]: Rolling back to ver13

### DISCUSSION THREAD

**Discussion by zaPure** [blakeblackshear/frigate#13015] [Ask A Question]
### What is your question:

Hi

My system auto-updated last night to 0.14 and is now no longer working.
Can I roll back to the previous version? I will only have time to fix it on the weekend. Away on work and use frigate to monitor that everything is ok at home. So this worrying.

I tried on my unraid docker template to revert it to the previous version by setting the respository from :stable to ghcr.io/blakeblackshear/frigate:0.13.2 and re-using my old config file. But I can't get it up and running.

Says my config is invalid and then adds 
version: 0.14 at the end of the config.

Is there anything else I'm missing

**Comment by mimainka** [comment]
Without the config and log i guess its kind of hard to tell whats the reason for this.
Could be: 
- did you create camera groups (wasnt there in 0.13)?
- did you create masks and zones in the 0.14 UI (they are now relative-values)?
- tls?
- why is it adding version 0.14 in the old config - is it really using the 0.13.2 image?

**Comment by zaPure** [comment]
Thank you for the reply
I originally couldn't get 0.14 up and running. 
I found some time today and got 0.14 working in its basic form. 
I need to spend some time and get hard the hardware acceleration working again., everything is running off the CPU.

**Comment by JoFranke** [comment]
I second that question - what is the proper way of going back to 0.13? I'm running bare metal standalone (no HA addon) and would like to go back to 0.13.

I guess its just copying back the 0.13 DB and config? TIA

**Reply by NickM-27** [reply]
Yes, backups of both were made

**Reply by mbronstein1** [reply]
Re: the backups, I tried to revert to 0.13.2 and use the backups, and I got an error that the database disk was malformed. I ended up needing to delete my db and let it re-create from scratch on the next docker compose up.

**Reply by mbronstein1** [reply]
Might've had something to do with the docker caching? Not sure.
